creator-node/src/utils.js

```diff
@@ -434,6 +435,26 @@ async function writeStreamToFileSystem (stream, expectedStoragePath, createDir =
   })
 }
 
+async function runShellCommand (command, args, logger) {
```

Comment:
this is great - do you mind adding a small comment explaining how it works above this function? bc i have a feeling i will need to use it later

Reply:
just added @hareeshnagaraj https://github.com/AudiusProject/audius-protocol/pull/1043/commits/18448694dc669ec3cf2c058084a46fd7ec1eaf5d